ElectronicObserver/Utility/SoftwareUpdater.cs

```diff
@@ -19,12 +21,20 @@ namespace ElectronicObserver.Utility;
 internal class SoftwareUpdater
 {
 	internal static string AppDataFolder =>
-		Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData) + @"\\ElectronicObserver";
+		Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "ElectronicObserver");
 
 	private static bool WaitForRestart { get; set; }
 	public static UpdateData CurrentVersion { get; set; } = new UpdateData();
 	public static UpdateData LatestVersion { get; set; } = new UpdateData();
 
+	private static Uri DataUpdateURL => new($"{Configuration.Config.Control.UpdateRepoURL}/update.json");
+
+	private static Uri TranslationUpdateURL => new($"{Configuration.Config.Control.UpdateRepoURL}/Translations/{DataAndTranslationManager.CurrentTranslationLanguage}/update.json");
+
+	private static string DataUpdateFile => $"{DataAndTranslationManager.WorkingFolder}\\update.json";
```

Comment:
use Path.Combine here for consistency

Reply:
[bd89801](https://github.com/gre4bee/ElectronicObserver/pull/161/commits/bd898013892d19b9fac30d74b7255cb7f766b8dd)